test.js

```diff
@@ -1,14 +1,55 @@
-var Validate = require('can-validate-validatejs');
+var makeValidator = require('can-validate-validatejs');
 var QUnit = require('steal-qunit');
+var isEmptyObject = require('can-util/js/is-empty-object/is-empty-object');
+var isArray = require('can-util/js/is-array/is-array');
 
-var validate;
+var constraints = {
+    age: {
+        numericality: {
+            message: 'should be a number'
+        }
+    },
+    name: {
+        presence: {
+            message: 'cannot be blank'
+        }
+    }
+};
 
-QUnit.module('Map Validate Plugin', {
-	setup: function(){
-		validate = new Validate();
-	}
+var invalidPerson = {
+    name: '',
+    age: 'hello'
+};
+
+var validPerson = {
+    name: 'Juan',
+    age: 35
+};
+
+QUnit.module('can-validate-validatejs');
+
+QUnit.test('makeValidator sets errors',function(){
+    var validateAge = makeValidator(constraints.age);
+    errors = validateAge(invalidPerson.age);
```

Comment:
Good catch. Jshint is there but minimally configured. I would think the default config would check for this.